ethexe/contracts/foundry.toml

```
@@ -1,4 +1,5 @@
 [profile.default]
+solc_version = "0.8.26"
```

pin it to latest version
```suggestion
solc_version = "0.8.28"
```

---

done